bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionRegistry.java

```diff
@@ -166,11 +196,11 @@ protected void removeConfigDescriptionProvider(ConfigDescriptionProvider configD
             List<ConfigDescriptionParameter> parametersWithOptions = new ArrayList<ConfigDescriptionParameter>(
                     parameters.size());
             for (ConfigDescriptionParameter parameter : parameters) {
-                parametersWithOptions.add(getConfigOptions(uri, parameter, locale));
+                parametersWithOptions.add(getConfigOptions(aliasedURI, parameter, locale));
             }
 
             // Return the new configuration description
-            return new ConfigDescription(uri, parametersWithOptions, parameterGroups);
+            return new ConfigDescription(aliasedURI, parametersWithOptions, parameterGroups);
```

Comment:
Hm, again, shouldn't you return a config description for the uri that was originally requested?